native/native_main.py

```diff
@@ -116,6 +115,13 @@ def handleMessage(message):
             file.write(message["content"])
         reply['content'] = filepath
 
+    elif cmd == 'env':
+        reply['content'] = getenv(message["var"], "")
+
+    # Gets the parent Firefox process's command line arguments
+    elif cmd == 'ffargs':
+        reply["content"] = handleMessage({"cmd": "run", "command": "ps -p " + str(os.getppid()) + " -o args="})["content"]
```

Comment:
I think this would be better in native_background.ts as `ps -p` is platform specific and I would like the messenger to be as cross platform as possible. Happy for os.getppid to live here, but it is probably used infrequently enough that we can just use `eval`.

Reply:
I see. I thought the distinction was "eval for user code and native stuff for our own" but your way of doing makes more sense.